CMakeLists.txt

```diff
@@ -7,8 +7,8 @@
 #
 # Build notes:
 #
-# * https://rtt.lanl.gov/redmine/projects/draco/wiki/
-# * https://rtt.lanl.gov/redmine/projects/draco/wiki/Common_Configure_Options
+# * https://re-git.lanl.gov/draco/draco/-/wikis/home
+# * https://re-git.lanl.gov/draco/draco/-/wiki/Common_Configure_Options
```

Comment:
Do we want to use re-git or asc-git in our documentation?

Reply:
For now, we need to use `re-git`.  `asc-git` is an alias but there are authentication issues related to forwarding requests from `asc-git` to `re-git`.